### PR TITLE
add docs

### PR DESCRIPTION
**Pull request by sstraatemans**
PR checklist:

* [ ] Test coverage for the proposed changes
* [ ] PR description contains example output from repl interaction or a snippet from unit test output
* [ ] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/kadena-io/pact-core/blob/master/CHANGELOG.md)

Additionally, please justify why you should or should not do the following:

* [ ] Confirm replay/back compat
* [ ] Benchmark regressions
* [ ] (For Kadena engineers) Run integration-tests against a Chainweb built with this version of Pact

### CODE REVIEW

File: docs/builtins/General/list-modules.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## list-modules
```
Comment (rsoeldner) [inline docs/builtins/General/list-modules.md:1]: is being dropped

File: docs/builtins/General/public-chain-data.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## public-chain-data
```
Comment (rsoeldner) [inline docs/builtins/General/public-chain-data.md:1]: also dropped?

File: docs/builtins/Keysets/read-keyset.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## read-keyset
```
Comment (rsoeldner) [inline docs/builtins/Keysets/read-keyset.md:1]: is in `General` atm

File: docs/builtins/Operators/round.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+## round
```
Comment (rsoeldner) [inline docs/builtins/Operators/round.md:1]: in general

File: docs/builtins/Repl/bench.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+## bench
```
Comment (rsoeldner) [inline docs/builtins/Repl/bench.md:1]: dropped

File: docs/builtins/Repl/env-chain-data.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+## env-chain-data
```
Comment (rsoeldner) [inline docs/builtins/Repl/env-chain-data.md:1]: not there atm

File: docs/builtins/Repl/env-dynref.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+## env-dynref
```
Comment (rsoeldner) [inline docs/builtins/Repl/env-dynref.md:1]: not there atm

File: docs/builtins/Repl/env-entity.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+## env-entity
```
Comment (rsoeldner) [inline docs/builtins/Repl/env-entity.md:1]: not implemented

File: docs/builtins/Repl/env-exec-config.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+## env-exec-config
```
Comment (rsoeldner) [inline docs/builtins/Repl/env-exec-config.md:1]: needs to be discussed

File: docs/builtins/Repl/env-chain-data.md (hunk 6 above)
Comment (jmcardon) [reply docs/builtins/Repl/env-chain-data.md:1]: This is definitely there.

**Comment by rsoeldner** [comment]
closing as #122 captures the content\builtins

**Pull request closed without merging**